cli/jobs/deepspeed/deepspeed-training/generate-yml.sh

```diff
@@ -26,7 +32,7 @@ outputs:
 compute: azureml:gpu-v100-cluster
```

Comment:
how many GPUs per node does this cluster have? If it has 4, then we should initialize the variable holding the process count per node with 4..

Reply:
It has 8, should we use all 8 here?